src/modules/theme/components/search/select.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, Output, EventEmitter, OnInit } from '@angular/core';
+
+export type SelectComponentInputType = { key: string; text: string; order: 'ascending' | 'descending' };
```

Comment:
It was actually that way before, but Diogo suggested changing it to a more generic 'Select' since that is what it really is, and I agree. 
But I do agree with you that, following that logic, the order should not be inside that component. 
I will think about how to extract that logic to the advanced review component.

Thanks!

Reply:
I agree, with this component to me more "generic" as Pablo mentioned!
And I agree that the responsibilities of the `select` component should be:

- Display a list of options provided by the parent (Input);
- Emit an event of the selected option (Output).

In this case, if it's ascending or descending, it is not the responsibility of the select `component` but the component that uses it.